### PR TITLE
[update] Deploy a Kubernetes Cluster Using kubeadm

### PR DESCRIPTION
**Pull request by Rajakavitha1**
_ update the commands
_ replace master with primary and worker with secondary

### CODE REVIEW

**Review by wildmanonline — CHANGES_REQUESTED** [review]
Kubernetes doesn't have the concept of *primary* and *secondary* nodes, at least in the official documentation. AFAIK, there is the concept of a *control plane* and *node* (*worker machine*, also called a *worker node*, but just *node* suffices as there is no other type of node.). The term *master* is deprecated (https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-master) and it seems like *worker node* has been generally replaced with just *node*

**Comment by Rajakavitha1** [comment]
> Kubernetes doesn't have the concept of _primary_ and _secondary_ nodes, at least in the official documentation. AFAIK, there is the concept of a _control plane_ (also called _control plane node_) and _node_ (_worker machine_, also called a _worker node_). The term _master_ is deprecated (https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-master) but it seems like _worker node_ is still used (alongside just _node_ or _worker machine_).

Hi @wildmanonline . Updated the word usage as you suggested. master to control plane and worker node remains as is.

File: docs/guides/kubernetes/deploy-kubernetes-cluster-using-kubeadm/index.md [hunk 1]
```diff
@@ -68,11 +68,11 @@ While kubeadm automates several cluster-provisioning tasks, there are other even
...
+A Kubernetes cluster consists of a control plane and worker nodes. The control plane hosts the *control plane*, which is the combination of all the components that provide it the ability to maintain the desired cluster state. This cluster state is defined by manifest files and the <abbr title="kubectl is a command line tool used to interact with the Kubernetes cluster.">kubectl</abbr> tool. While the control plane components can be run on any cluster node, it is a best practice to isolate the control plane on its own node and to run any application containers on a separate worker node. A cluster can have a single worker node or up to 5000. Each worker node must be able to maintain running containers in a Pod and be able to communicate with the control plane.
```
Comment (andystevensname) [inline docs/guides/kubernetes/deploy-kubernetes-cluster-using-kubeadm/index.md:71]: "The control plane hosts the *control plane*" should be updated

**Review by andystevensname — CHANGES_REQUESTED** [review]

File: docs/guides/kubernetes/deploy-kubernetes-cluster-using-kubeadm/index.md (hunk 1 above)
Comment (Rajakavitha1) [reply docs/guides/kubernetes/deploy-kubernetes-cluster-using-kubeadm/index.md]: Done!!!

**Review by andystevensname — APPROVED** [review]

**Review by andystevensname — APPROVED** [review]

**Review by andystevensname — APPROVED** [review]

**Review by wildmanonline — APPROVED** [review]

**Pull request merged**